ansible/monitoring.yml

```diff
@@ -44,7 +44,7 @@
         init_config:
         instances:
           - name: API
-            url: "{{ api_base_uri|default('127.0.0.1') }}/top"
+            url: "{{ api_base_uri|default('http://127.0.0.1:3013/v2') }}/top"
```

Comment:
Could default for `api_base_uri` be removed from here?

Reply:
Not sure what you mean exactly, could you please elaborate with more details ?

Reply:
If you mean to move the default to the api_base_uri definition in `ansible/group_vars/tag_role_epoch/vars.yml`, it would be rather complex task because of the vaults. Right now we intentionally don't tag the `image.yml` playbook run with `tag_role_epoch`

To be able to do that we should first take out the secrets in `ansible/group_vars/tag_role_epoch/vault.yml` to the environment somehow.